src/test/key_tests.cpp

```diff
@@ -220,4 +221,47 @@ BOOST_AUTO_TEST_CASE(key_key_negation)
     BOOST_CHECK(key.GetPubKey().data()[0] == 0x03);
 }
 
+static CPubKey UnserializePubkey(const std::vector<uint8_t>& data)
+{
+    CDataStream stream{SER_NETWORK, INIT_PROTO_VERSION};
+    stream << data;
+    CPubKey pubkey;
+    stream >> pubkey;
+    return pubkey;
+}
+
+static unsigned int GetLen(unsigned char chHeader)
+{
+    if (chHeader == 2 || chHeader == 3)
+        return CPubKey::COMPRESSED_SIZE;
```

Comment:
nit: Either single-line or with `{   }`

Reply:
I thought about changing that but I decided not to because it's a straight copy out of the CPubKey class (a private function)

Reply:
Yeah, no worries its just the tests.